services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
It accepts Optional[str] because by default it accepts None
Fix in other places(only your code)